app/metrics.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint
```

Comment:
On Brief Responses FE we put this in `app/metrics/__init__.py` while the APIs use `app/metrics.py` - it probably doesn't matter which one we use, but should we pick one for consistency?

Reply:
Yeah, I think `app/metrics.py` is more pythonic.

Reply:
https://github.com/alphagov/digitalmarketplace-brief-responses-frontend/pull/117